web/tests/simple.spec.ts

```diff
@@ -0,0 +1,15 @@
+import {test, expect} from "../playwright";
+
+test("select version query", async ({page, uiClass, gelClient}) => {
+  await page.goto("main/editor");
+
+  await page.locator(".cm-content").fill("select sys::get_version_as_str()");
+
+  await page.getByRole("button", {name: "Run"}).click();
+
+  await expect(uiClass("inspector_scalar_string")).toHaveText(
+    await gelClient.queryRequiredSingle<string>(
+      `select sys::get_version_as_str()`
+    )
+  );
+});
```

Comment:
in this test u are just checking if what is shown in the UI is what is returned by gelClient?

Reply:
Yes, I think this test was just meant as a sanity check.